<?xml version="1.0"?>
<launch>

	<arg name="dataset" value="test12"/>

    <rosparam param="/uwb/nodesId"> [100, 101, 102, 103, 201]</rosparam>
    <rosparam param="/uwb/nodesPos">[3, 3, 0.65, 3, -3, 0.25, -3, -3, 0.25, -3, 3, 0.25, 0, 0, 1.0]</rosparam>

    <node name="rosbag_play" pkg="rosbag" type="play" output="screen" args="$(find localization)/bag/$(arg dataset).bag"/>
  
    <node pkg="localization" type="localization_node" name="localization_node" output="screen">
		<remap from="~/pose" to="/ni_slam_node/incremental_pose_cov"/>
		<remap from="~/range" to="/uwb_endorange_info"/>
		<remap from="~/imu" to="/imu/data"/>
		<!-- <remap from="~/twist" to="/viconxbee_node/mocap/twist"/> -->
		<param name="localization/filename_prefix" value="$(find localization)/bag/$(arg dataset)" />
    </node>

    <node pkg="localization" type="pose2file_node" name="pose2file_node" output="screen"/>

</launch>

